src/cli/commands/create.js

```diff
@@ -16,17 +16,50 @@ export function hasWrapper(commander: Object, args: Array<string>): boolean {
   return true;
 }
 
+const coerceName = (name = '') => (name === '' ? 'create' : `create-${name}`);
+const coerceScope = (scope = '') => (scope === '@' ? '' : scope);
+const coerceFullName = (scope, name) => [coerceScope(scope), coerceName(name)].filter(Boolean).join('/');
+
+/**
+ * # Tests
+ *
+ * ## basic
+ * parseBuilderName('name').packageName === 'create-name'
+ * parseBuilderName('@scope/name').packageName === '@scope/create-name'
+ *
+ * ## not adding "-" if name is empty
+ * parseBuilderName('@scope/').packageName === '@scope/create'
+ * parseBuilderName('@scope').packageName === '@scope/create'
+ *
+ * ## edge cases
+ * parseBuilderName('@/name').packageName === 'create-name'
+ * parseBuilderName('/name').packageName === 'create-name'
+ * parseBuilderName('@/').packageName === 'create'
```

Comment:
I'd prefer if you could put this in a file called `__tests__/commands/create.js` (you can just export the `parseBuilderName` symbol and require the file), so that they are actually ran. Otherwise the comment can just be removed, since it will never be run.

Reply:
okay, will do

Reply:
added tests